config/webpack/webpack.config.js

```diff
@@ -55,7 +55,7 @@ module.exports = [
 		{
 			entry: languages.reduce( ( memo, language ) => {
 				const name = ( language === "_default" ) ? "default" : language;
-				memo[ name ] = "./node_modules/yoastseo/src/languageProcessing/languages/" + language + "/Researcher";
+				memo[ name ] = "./node_modules/yoastseo/build/languageProcessing/languages/" + language + "/Researcher";
```

Comment:
When we create bundled entries for language specific Researcher in Free, I change it so that we use the build code from `yoastseo`.

Reply:
This means that when building the plugin, we should also build the package.